src/components/ToolMenu/Draw/Attributions/index.tsx

```diff
@@ -0,0 +1,216 @@
+import React, {
+  useEffect, useState
+} from 'react';
+
+import {
+  MinusCircleOutlined, PlusOutlined
+} from '@ant-design/icons';
+import {
+  Button, Drawer, Form, Divider, Row
+} from 'antd';
+import _cloneDeep from 'lodash/cloneDeep';
+import OlFeature from 'ol/Feature';
+
+import { useTranslation } from 'react-i18next';
+
+import {
+  useMap
+} from '@terrestris/react-geo/dist/Hook/useMap';
+
+import './index.less';
+import AttributionRow from './AttributionRow';
+
+export type AttributionDrawerProps = {
+  openAttributeDrawer: boolean;
+  onClose: (open: boolean) => void;
+};
+
+const AttributionDrawer: React.FC<AttributionDrawerProps> = ({
+  openAttributeDrawer,
+  onClose
+}) => {
+  const [selectedFeature, setSelectedFeature] = useState<OlFeature>();
+  const [render, setRender] = useState<boolean>(true);
```

Comment:
Is this state used (effectively)?

Reply:
Currently this state force renders the drawer as soon as an element is deleted. If the state is missing, the element would be successfully deleted but only disappear after opening the drawer again. Alternatively this could be moved in a useEffect-hook also.

Reply:
Have you tried to clone the properties before removing the key? Like so (references line [106](https://github.com/terrestris/shogun-gis-client/pull/660/files#diff-f987335093293a4af957d2a42222e048b7c84be3c0b8c138a305a77f9ad7ac5aR106) and following):

```
const updatedProperties = {...currentProperties};
// OR
const updatedProperties = structuredClone(currentProperties);

delete updatedProperties[keyToRemove];

(…)
```

Reply:
This is adjusted.